pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -864,6 +910,19 @@ func (oe *operationExecutor) generateVerifyControllerAttachedVolumeFunc(
 			return nil
 		}
 
+		if !volumeToMount.ReportedInUse {
+			// If the given volume has not yet been added to the list of
```

Comment:
Should we add a TODO somewhere that we could force an immediate flush of our cache?